regression/scripts/ramjet_network/ramjet_network.py

```diff
@@ -0,0 +1,228 @@
+    
+    # instantiate
+    ram = SUAVE.Components.Energy.Converters.Ram()
+    ram.tag = 'ram'
+    
+    # add to the network
+    ramjet.append(ram)
+
+    # ------------------------------------------------------------------
+    #  Component 2 - Inlet Nozzle
+    
+    # instantiate
+    inlet_nozzle = SUAVE.Components.Energy.Converters.Compression_Nozzle()
+    inlet_nozzle.tag = 'inlet_nozzle'
+    
+    # setup
+    inlet_nozzle.polytropic_efficiency      = 1.0
+    inlet_nozzle.pressure_ratio             = 1.0
+    inlet_nozzle.compressibility_effects    = True
+    
+    # add to network
+    ramjet.append(inlet_nozzle)
+      
+    # ------------------------------------------------------------------
+    #  Component 3 - Combustor
+    
+    # instantiate    
+    combustor = SUAVE.Components.Energy.Converters.Combustor()   
+    combustor.tag = 'combustor'
+    
+    # setup
+    combustor.efficiency                = 1.0 
+    combustor.turbine_inlet_temperature = 2400.
+    combustor.pressure_ratio            = 1.0
+    combustor.fuel_data                 = SUAVE.Attributes.Propellants.Jet_A()  
+    combustor.rayleigh_analyses         = True
+    
+    # add to network
+    ramjet.append(combustor)
+
+    # ------------------------------------------------------------------
+    #  Component 4 - Core Nozzle
+    
+    # instantiate
+    nozzle = SUAVE.Components.Energy.Converters.Supersonic_Nozzle()   
+    nozzle.tag = 'core_nozzle'
+    
+    # setup
+    nozzle.polytropic_efficiency = 1.0
+    nozzle.pressure_ratio        = 1.0   
+    
+    # add to network
+    ramjet.append(nozzle)
+
+    # ------------------------------------------------------------------
+    #  Component 5 - Thrust
+    
+    # instantiate
+    thrust = SUAVE.Components.Energy.Processes.Thrust()       
+    thrust.tag ='thrust'
+    
+    # setup
+    thrust.total_design = ramjet.number_of_engines*169370.4652 * Units.N
+    
+    # add to network
+    ramjet.thrust = thrust    
+
+    #size the ramjet
+    ramjet_sizing(ramjet,0.8,10000.0)
+    
+    print "Design thrust ",ramjet.design_thrust
+    print "Sealevel static thrust ",ramjet.sealevel_static_thrust
+    
+    results_design     = ramjet(state_sizing)
+    results_off_design = ramjet(state_off_design)
+    F                  = results_design.thrust_force_vector
+    mdot               = results_design.vehicle_mass_rate
+    F_off_design       = results_off_design.thrust_force_vector
+    mdot_off_design    = results_off_design.vehicle_mass_rate
+    
+    #Specify the expected values
+    expected = Data()
+    
+    expected.thrust = 3056962.9065545597
+    expected.mdot   = 203.11136550916942
+    
+    #error data function
+    error =  Data()
+    
+    error.thrust_error = (F[0][0] -  expected.thrust)/expected.thrust
+    error.mdot_error   = (mdot[0][0] - expected.mdot)/expected.mdot
+    print error
+    
+    for k,v in error.items():
+        assert(np.abs(v)<1e-4)    
```

Comment:
Any tolerance used greater than 1e-6 needs to be justified and reason should be recorded in a comment.